test/lib/ansible_test/_data/completion/docker.txt

```diff
@@ -1,3 +1,4 @@
+alpine3 name=quay.io/ansible/alpine3-test-container:1.18.0 python=3.8
```

Comment:
I've been keeping this file sorted, except for the default entries being at the top, so I'd put this just before centos6.